pkgs/development/tools/vagrant/default.nix

```diff
@@ -88,7 +92,14 @@ in buildRubyGem rec {
     cp -av contrib/bash/completion.sh $out/share/bash-completion/completions/vagrant
   '' +
   lib.optionalString withLibvirt ''
-    substitute ${./vagrant-libvirt.json.in} $out/vagrant-plugins/plugins.d/vagrant-libvirt.json \
+    substitute ${./vagrant-plugin.json.in} $out/vagrant-plugins/plugins.d/vagrant-libvirt.json \
+      --subst-var-by pname libvirt \
+      --subst-var-by ruby_version ${ruby.version} \
+      --subst-var-by vagrant_version ${version}
+  '' +
```

Comment:
Can you create a small helper. I consider adding vagrant-sshfs and reuse that code.